src/Microsoft.AspNet.Mvc.Core/ModelBinders/BodyModelBinder.cs

```diff
@@ -69,10 +69,10 @@ public class BodyModelBinder : BindingSourceModelBinder
             {
                 model = GetDefaultValueForType(bindingContext.ModelType);
```

Comment:
Not specific to this PR, but is this value actually used?
Same remark for the `_bodyValidationExcludeFiltersProvider`, which is imported but not used.